locale/ru-ru/dialog/daily/daily-weather-loation.dialog

```diff
@@ -1,3 +1,3 @@
-{{day}} будет {{condition}}, с максимумом {{temp_max}} и минимумом {{temp_min}}
-{{day}}, {{location}} самая высокая температура будет {{temp_max}} и самая низкая - {{temp_min}}, с {{condition}} погодными условиями
-Прогноз погоды для {{day}} {{temp_max}} как высокая температура и {{temp_min}} как низкая
+{day} будет {condition}, температура поднимется до {temp_max} и опустится до {temp_min}
```

Comment:
The variables `temp_max`/`temp_min` have to be swapped with `high_temperature`/ `low_temperature`. (in general)

Reply:
Good catch! It was a file with a misspelled name. Apparently, that's why the old variables were still there. I deleted it completely.